cmd/machine-api-operator/start.go

```diff
@@ -75,6 +81,16 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 }
 
 func startControllers(ctx *ControllerContext) error {
+	kubeClient := ctx.ClientBuilder.KubeClientOrDie(componentName)
+
+	eventRecorderScheme := runtime.NewScheme()
+	osconfigv1.Install(eventRecorderScheme)
+
+	eventBroadcaster := record.NewBroadcaster()
```

Comment:
what's the motivation behind moving this here?

Reply:
to make possible to pass fake recorder under the unittest